README.md

```diff
@@ -1,3 +1,17 @@
+# Hierarchical deterministic password manager
+
+## Compilation Instructions
+
+**MACOS (10.15)**  
+This project requires an `ncurses` implementation, you can use homebrew to get one:
+```bash
+$ brew install ncurses
+```
+Then it's time to compile with:
+```bash
+$ cmake .
```

Comment:
The standard (can actually be outside macos) would be to
```
mkdir build
cd build
cmake ..
make
```

Reply:
fair enough, I'll update.